### ISSUE TITLE
Update vertical CRS to improved CRS

### ISSUE THREAD

**Issue by HerzovanderWal** [INSPIRE-MIF/technical-guidelines#27]
For the provision of elevation data, INSPIRE foresees the use of EVRF2000, EPSG:5730. 

Since then, this concept has been updated twice, adding the following elevation codes with adjusted vertical Datum:
•	EVRF2007 height    EPSG:5621
•	EVRF2019 height    EPSG:9389
•	EVRF2019 mean-tide height    EPSG:9390

We would kindly request the addition of these EPSG Codes to the list of CRS applicable within INSPIRE.

## Addressed TG
https://inspire.ec.europa.eu/id/document/tg/rs
https://inspire.ec.europa.eu/id/document/tg/gg
https://inspire.ec.europa.eu/id/document/tg/el

## Impact on INSPIRE validator
The CRS-s should become valid values.

## Impact on INSPIRE code lists
The CRS-s should become valid values.

## Change proposer
Herzo van der Wal
Data-Manager at Ministry of Infrastructure and Water Management
Rijkswaterstaat (directed to this site by Geonovum)
Responsible for INSPIRE-services on the thema Elevation and more

## References
https://github.com/codefornl/INSPIRE-Coverages

**Comment by sMorrone** [comment]
@HerzovanderWal 
your request is linked to the amendment to COMMISSION REGULATION 1089/2010 which, inter alia, implements the changes (already agreed at MIG level) related to the setup of a Coordinate Reference System (CRS) registry and control body to manage the proposal of additional CRS under the governance of the MIG. Amendment is foreseen by end of Q2/2022. The CRS registry should then be set up, including governance structures (control body, etc.) and, once this is done, requests for the additional CRS can be submitted, and, if accepted by the control body, implemented (therein including relevant update of the INSPIRE artefacts). 
For more details see https://github.com/INSPIRE-MIF/helpdesk/issues/5 and https://github.com/INSPIRE-MIF/helpdesk/issues/33#issuecomment-856750683 , or query the[ INSPIRE general hepdesk](https://github.com/INSPIRE-MIF/helpdesk) by **CRS registry** label

**Comment by jescriu** [comment]
Dear @sMorrone, 

I participated as editor of the Elevation Technical Guidelines (EL TG) and I think the proposal from Herzo is not exactly a request for adding additional CRSs. Perhaps it could be processed in a more flexible manner - To be further analysed and confirmed.

Al the EPSG codes proposed by @HerzovanderWal refers to the European Vertical Reference System (EVRS), which is the CRS adopted by INSPIRE for providing Elevation values on land in the Continental Europe, as per Annex II, Section 1.3.3 point 2 of the [IR on Interoperability of Spatial Datasets and Services](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:02010R1089-20141231&qid=1643047585427&from=en) (COMMISSION REGULATION (EU) No 1089/2010).

In my personal view, any realization (or reference frame) of the EVRS should be accepted for use in INSPIRE: at least either EPSG:5730 and EPSG:5621. 

**As illustration** - At the time of writing the [EL TG](https://inspire.ec.europa.eu/file/1530/download?token=pq85sbLG) (published at the end of 2013) the EVRF 2007 realization of EVRS was already recommended – See below Recommendation 15:
![image](https://user-images.githubusercontent.com/22743378/150831630-75751b3e-ea56-4ac2-a98a-1ae6d5067855.png)

The [Coordinate reference systems Technical Guidelines](https://inspire.ec.europa.eu/file/1726/download?token=3OGur2Ln) (RS TG) includes a TG Requirement with the CRS identifiers to be used for referring the default CRSs – See TG Requirement 1 and Table 1 below: 
![image](https://user-images.githubusercontent.com/22743378/150832502-5116f6ec-0ff1-4393-8c08-7558c395c7c7.png)

Certainly only EPSG:5730 (the EVRF2000 realization of EVRS) is included for EVRS in the table above. But note that the document was published at the beginning of 2014, when even there was not any EPSG URI available for the EVRF2007, realization of EVRS recommended by the EL TG.

**Summing up**:
- The EL TG already acknowledges that either EVRF2000 (EPSG:5730) or EVRF2007 (EPSG:5621) can be used (the latter is recommended). To check with the experts if the same applies for the other EPSGs proposed by Herzo.

If this is the case:
- Table 1 from the RS TG could be updated accordingly, introducing those EPSG codes that can be accepted, without having to introduce changes to the COMMISSION REGULATION (EU) No 1089/2010. 
- The coordinate reference systems tests in the INSPIRE Reference Validator could also be updated for accepting these EPSG codes.

**Comment by sMorrone** [comment]
Subgroup meeting on 24.03.2022:
the Subgroup approved this change proposal.

**Comment by HerzovanderWal** [comment]
Great! Thank you

**Comment by jescriu** [comment]
Please, have a look at https://github.com/INSPIRE-MIF/helpdesk/blob/main/crs.md 
The CRSs endorsed should be included as part of the ones accepted by the INSPIRE Reference Validator.
Once this is done I assume this issue could be closed.

**Comment by fabiovinci** [comment]
The Data Specification on Geographical Grid Systems does not contain references and/or tables related to vertical CRSs, so no changes will be applied to this TG.
